### DISCUSSION TITLE
Error 404 with BCI package

### DISCUSSION THREAD

**Discussion by YanisChrys** [IBAHCM/RPiR#315] [Archive] [answered]
When I was trying to do devtools::install() on the Diversity package and load my BCI package as a dependency I was getting a 404 Error asking me if I misspelled the package name or if I have permission to use it. I followed all the right steps to load it but it let me install the Diversity package only after I created a personal access token. Will this be a problem for anyone else trying to run the package?

**Comment by soniamitchell** [comment, marked as answer]
Hi @YanisChrys, the PAT is required because your BCI package is private and RStudio needs permissions, don’t worry. I think most people have a PAT token already (I thought everyone did), but if they don’t then we can advise them :)
